.goreleaser.yml

```diff
@@ -28,6 +28,8 @@ builds:
     ldflags:
       - "{{ .Env.COMMON_LDFLAGS }}"
       - -X github.com/chainloop-dev/chainloop/app/cli/cmd.Version={{ .Version }}
+      - -X main.posthogAPIKey={{ .Env.POSTHOG_API_KEY }}
```

Comment:
Let's make sure we have a page documenting this change that goes up in the same Chainloop release :)

Reply:
Yes, I still have pending add a documentation page.